aqueduct/gateway/tests/test_endpoints.py

```diff
@@ -186,15 +252,18 @@ def test_chat_completion_base64_file_input(self):
                 }
             ],
             "max_completion_tokens": 50,
+            "user_id": 42,
```

Comment:
A test with email might be nice as well for the upcoming #71. It should work since spaces also work (`test_embeddings_with_user_id_in_body`) Also a separate test class for user_id might be helpful here as well since `test_chat_completion_base64_file_input` now also tests "user_id works with int value" and `test_embeddings_with_user_id_in_body` tests "user_id works with spaces". Gets quite confusing quickly. See other comment. This is the perfect candidate for a parameterized test with all "edge cases" in one place instead of it being scattered throughout different tests with names that describe something else.